README.md

```diff
@@ -292,6 +292,35 @@ auth0GuardianJS.start(function(err, transaction) {
   //...
 });
 ```
+### auth0GuardianJS.resume(options, transactionState, callback) 
+This continues a transaction saved by `transaction.serialize()`. The option parameter provides 
+the final user the opportunity to specify which kind of `transport` to use. That been:
+
+- `socket`: a socket.io transport
+- `pooling`: a pooling transport.
```

Comment:
"The option parameter" -> "The options parameter"

"That been:" -> "Options include:"